### DISCUSSION TITLE
help !! I can't send message

### DISCUSSION THREAD

**Discussion by mofisher** [okhiroyuki/node-red-contrib-line-notify#67] [Q&A] [answered]
The error message is as followed.  I don't know how to fix it.  
[flows (2).txt](https://github.com/okhiroyuki/node-red-contrib-line-notify/files/6235516/flows.2.txt)

(node:13047) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'data' of undefined
    at /usr/lib/node_modules/node-red-contrib-line-notify/nodes/line-notify.js:140:45
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:13047) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:13047) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by mofisher** [comment, marked as answer]
Solved. 
It's my network problem. After set DNS, it works now.

**Reply by okhiroyuki** [reply]
That's good to know.
